cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -348,6 +351,43 @@ describe('Unit tests for add_disaster page', () => {
   });
 });
 
+/**
+ * Set's local cookie to point to disaster with the given layers.
+ * @param {Array<Object>} layers
+ */
+function setDisasterAndLayers(layers) {
+  const currentDisaster = '2005-fall';
+  disasterData.set(currentDisaster, {layers: layers});
+  window.localStorage.setItem('disaster', currentDisaster);
+}
+
+/**
+ * Function that tests the save method works.
+ * @param {Function} fxn save function
+ * @param {string} property
+ * @param {Object} input DOM object save from which is pulling value
+ * @param {*} afterVal
```

Comment:
Nit: are we using * for param types?

Reply:
I don't think we have up until now, but this feels like a * situation - right now it's being used for a boolean, string and list.